modules/nf-core/gprofiler2/gost/main.nf

```diff
@@ -0,0 +1,30 @@
+process GPROFILER2_GOST {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "conda-forge::r-ggplot2=3.4.3 conda-forge::r-gprofiler2=0.2.2"
```

Comment:
I am not sure how double conda envs are standardised in nf-core, so somebody else should have to check if containers are OK here. And/or if we need to move these to the environment.yml file in the main folder as well.
Would need to change this line here to:
conda "${moduleDir}/environment.yml"

and then have an environment.yml file:
channels:
  - conda-forge
  - bioconda
  - defaults
dependencies:
  - conda-forge::r-ggplot2=3.4.3
  - conda-forge::r-gprofiler2=0.2.2

Reply:
Ou, I did not notice nf-core had switched to an env.yml! Done!